js/default.js

```diff
@@ -81,6 +88,16 @@ var terminal = {
                     return callback(data);
                 });
             },
+            monitor: function(command) {
+                var every = "10";
```

Comment:
10 is just the default value, it checks every 10 seconds if you don't provide a value.